lib/ExtUtils/MM_Unix.pm

```diff
@@ -2227,6 +2228,7 @@ sub init_xs {
 		my ($v, $d, $f) = File::Spec->splitpath($ext);
 		my @d = File::Spec->splitdir($d);
 		shift @d if defined $d[0] and $d[0] eq 'lib';
+        pop @d if $d[$#d] eq '';
```

Comment:
Seems like the whitespace isn't correct here

Reply:
Apologies for that. I've added a whitespace commit.